config/manager/manager.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
```

Comment:
we should update this so it doesn't overlap

Reply:
Good catch, will do